leakage-test/leakage-test_instance.json

```diff
@@ -0,0 +1,74 @@
+{
```

Comment:
Can you describe in general what this ontology is used for?
A small use case would be good to understand it better.
Maybe you can add a readme to describe it there.

Reply:
a short description would really help to understand the use case of leakage-test.

Reply:
It's an ontology for the parameters of a leakage test done for a control unit.